lib/stripe/list_object.rb

```diff
@@ -96,6 +106,8 @@ def next_page(params = {}, opts = {})
     # This method will try to respect the limit of the current page. If none
     # was given, the default limit will be fetched again.
     def previous_page(params = {}, opts = {})
+      return self.class.empty_list(opts) unless has_more
```

Comment:
This was built into `next_page`, but apparently never added to `previous_page` which caused problems when I tried to actually use it for pagination.